docs/installing/vagrant.md

```diff
@@ -31,21 +67,27 @@ Ubuntu 12.04 LTS that contains everything you need to work on Alaveteli.
             $ git submodule update --init
 
 2.  Create the Vagrant VM. This will provision the system and can take some time
-    &mdash; sometimes as long as 20 minutes.
+    — sometimes as long as 20 minutes. Vagrant will download the files it
+    needs, so the first time you do this, you must be online for this to work.
 
             $ vagrant --no-color up
 
-3.  You should now be able to log in to the Vagrant guest OS with `ssh` and run
-    the test suite:
+    When the VM is ready, you'll see a message like `Machine booted and ready`,
+    and control will be back at your command prompt.
+    
+3.  Once the machine is up, you can log in to it by doing <code>vagrant&nbsp;up</code>
+    (from the same directory that contains the `VagrantFile`, so if you've
+    just run `up` you're already in the right place). This will log you into
+    the VM as the `vagrant` user. Immediately change to the `alaveteli`
+    directory, because you need to be there when issuing any of the admin or
+    rake tasks:
 
             $ vagrant ssh
 
             # You are now in a terminal on the virtual machine
             $ cd /home/vagrant/alaveteli
```

Comment:
note: the vagrant new session message has been updated and we currently suggest the simpler `cd alaveteli` as a quicker way of getting to the same place

current text:

<pre>
Welcome to your Vagrant-built virtual machine.
To start your alaveteli instance:
* cd alaveteli
* bundle exec rails server
</pre>